main/go.mod

```diff
@@ -10,234 +12,530 @@ replace github.com/brave-intl/bat-go/libs => ../libs
 
 replace github.com/brave-intl/bat-go/tools => ../tools
 
+replace github.com/pires/go-proxyproto v1.0.0 => github.com/pires/go-proxyproto v0.8.0
```

Comment:
Note: Here and below, this is necessary because one of the Hashicorp's dependencies listed `github.com/pires/go-proxyproto v1.0.0` in their `go.mod` file, but this version never existed. [The latest available version](https://github.com/pires/go-proxyproto/releases) is `v0.8.0`. So this `replace` directive makes it work.